src/main.rs

```diff
@@ -67,11 +67,12 @@ quick_main!(run);
 
 fn run() -> cli::error::Result<()> {
 	let version = VersionInfo {
+		name: "Parity Polkadot",
 		commit: vergen::short_sha(),
 		version: env!("CARGO_PKG_VERSION"),
 		executable_name: "polkadot",
-		author: "Parity Team <admin@parity.io>",
-		description: "Polkadot Node Rust Implementation",
+		author: "Parity Technologies <admin@parity.io>",
```

Comment:
I intentionally didn't change this `author` as it is only passed on to [`AppDirs`](https://docs.rs/app_dirs/1.2.1/app_dirs/) and according to theirs docs, _some_ OS include those in the folder as well, meaning you'd have to rename even more folders to upgrade - probably on Windows and MacOS. So the docs need updating. But Considering that we aren't displaying this anywhere with this PR, I think it is easier to keep as is with the simpler upgrade instructions.